### PR TITLE
feat(route): add nCoV2019

### PR DESCRIPTION
**Pull request by QieSen**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/coronavirus/tg-nCoV2019/广东
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/coronavirus/tg-nCoV2019.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+module.exports = async (ctx) => {
+	const district = ctx.params.district;
+	const queryUrl = district ? `https://t.me/s/nCoV2019?q=%23${encodeURIComponent(district)}` : `https://t.me/s/nCoV2019`;
+	const response = await got.get(queryUrl);
+	const $ = cheerio.load(response.data);
+	const list = $('.tgme_widget_message_bubble');
+	
+	if (list.length === 0 && $('.tgme_channel_history').length === 0) {
+        throw `Unable to fetch message feed from this channel. Please check this URL to see if you can view the message preview: https://t.me/s/${username}`;
```
Comment (nczitzk) [inline lib/v2/coronavirus/tg-nCoV2019.js:14]:
```suggestion
        throw 'Unable to fetch message feed from this channel. Please check this URL to see if you can view the message preview: https://t.me/s/nCoV2019';
```

**Review by nczitzk — CHANGES_REQUESTED** [review]
Well done except for this 😊

File: lib/v2/coronavirus/tg-nCoV2019.js (hunk 1 above)
Comment (vlcheng) [reply lib/v2/coronavirus/tg-nCoV2019.js:14]: Thanks, will be more attentive next time 😉

**Comment by TonyRL** [comment]
Any reasons why reinvent the wheel?
You can get similar output from `/telegram/channel/nCoV2019/searchQuery=%23广东`

**Comment by vlcheng** [comment]
@TonyRL 
If use /telegram/channel/nCoV2019, we will get such a title: 

> #疫情通报 #紧急寻人 #内蒙古 #Neimenggu #应急处置 #核酸检测 【#赤峰 市 #松山 区关于做好突发疫情应对工作的紧急通告】 2022年4月10日，赤峰市松山区在“愿检... 

The route only intercepts part of the main body text, and won't adapt to the message structure of this channel. 

If use this route, the title will be optimized:

> #赤峰 市 #松山 区关于做好突发疫情应对工作的紧急通告 // 划定“三区”范围 // 赤峰市关于组织开展中心城区全员核酸检测的通告

This route will parse title from "【】" in the body, and splice multiple titles together (if any).

If a frequently used channel has its stable message structure, why not consider configuring a separate route. 🙂

**Review by TonyRL — COMMENTED** [review]
Please lint your code

**Comment by DIYgod** [comment]
猜测这个路由已经没什么用了，所以本 pr 暂时关闭，如果还有疑问可以随时在下面继续讨论

**Pull request closed without merging**